# exceptions.py
class CannotConnect(Exception):
    """Impossible de se connecter Ã  l'API Vanmarcke"""

class InvalidAuth(Exception):
    """Identifiants invalides"""
